gtsam/hybrid/HybridGaussianFactorGraph.cpp

```diff
@@ -355,8 +355,9 @@ static std::shared_ptr<Factor> createHybridGaussianFactor(
       auto hf = std::dynamic_pointer_cast<HessianFactor>(factor);
       if (!hf) throw std::runtime_error("Expected HessianFactor!");
       // Add 2.0 term since the constant term will be premultiplied by 0.5
-      // as per the Hessian definition
-      hf->constantTerm() += 2.0 * conditional->logNormalizationConstant();
+      // as per the Hessian definition,
+      // and negative since we want log(k)
```

Comment:
Hmmm, should this method be be negLogProbabilities as well?

Reply:
Done